projects/core/src/user/occ/user.service.spec.ts

```diff
@@ -231,4 +232,28 @@ describe('OccUserService', () => {
       mockReq.flush('');
     });
   });
+
+  describe('reset password: ', () => {
+    it('should be able to reset a new password', () => {
+      const token = 'test token';
+      const newPassword = 'new password';
+
+      service
+        .resetPassword(token, newPassword)
+        .subscribe(result => expect(result).toEqual(''));
```

Comment:
We shouldn't have assertions in `subscribe` block, in case it doesn't execute

Reply:
I can modify it, but if you check all our tests of occ calls, you will find we have "expect" inside "subscribe". There is `mockReq.flush()`, so the subscribe will be execute.

Reply:
Ok, makes sense in that case